patches/chrome-browser-resources-bookmarks-reducers.js.patch

```diff
@@ -12,12 +12,12 @@ index 17ca9e18c8f4635aac2db849cb77886a4c7390dc..184e2f993812d6da4e09890336b2815e
  
      // Change node's parent.
      nodeModifications[id] =
-@@ -414,7 +417,7 @@ cr.define('bookmarks', function() {
+@@ -424,7 +427,7 @@ cr.define('bookmarks', function() {
          return FolderOpenState.openFolderAndAncestors(
              folderOpenState, nodes[action.id].parentId, nodes);
        case 'move-bookmark':
--        if (!nodes[action.id].children)
+-        if (!nodes[action.id].children) {
```

Comment:
Next line needs {, or remove the closing } line.

Reply:
This is fixed.